plugins/api/src/atomic_apply_acl_operations.cpp

```diff
@@ -496,8 +496,19 @@ namespace
 
         rodsLog(LOG_DEBUG, "Checking if user has permission to modify permissions ...");
 
-        if (!ic::user_has_permission_to_modify_acls(*_comm, db_conn, db_instance_name, object_id)) {
-            rodsLog(LOG_ERROR, "User not allowed to modify ACLs [logical_path=%s, object_id=%d]", logical_path.data(), object_id);
+        if (const auto iter = input.find("admin_mode"); iter != std::end(input) && iter->get<bool>()) {
+            if (!irods::is_privileged_client(*_comm)) {
+                const auto msg = fmt::format("User not allowed to modify ACLs [logical_path={}, object_id={}]. "
```

Comment:
consider...

"User not" -> "User [{}] not"

and fill it with full user#zone

Reply:
Yeah, I couldn't decide on whether that should be included because of the log message being referenced in the review comment below.

Will change. It's better to provide more information in general.

Reply:
Done